pos_product_available_negative/static/description/index.html

```diff
@@ -0,0 +1,124 @@
+<section class="oe_container">
+    <div class="oe_row oe_spaced">
+        <div class="oe_span12">
+            <h2 class="oe_slogan">Restriction a sale of out-of-stock products</h2>
+            <h3 class="oe_slogan">Only a supervisor can approve a POS order with a product which is out of stock</h3>
+        </div>
+    </div>
+</section>
+
+<section class="oe_container">
+    <div class="oe_row oe_spaced">
+        <div class="oe_span12">
+            <p class="oe_mt32">
+                The module checks whether an order contains products with not positive quantity.
+            </p>
+        </div>
+        <div class="oe_span6">
+            <div class="oe_demo oe_picture oe_screenshot">
+                <img src="screenshot-1.png"/>
+            </div>
+        </div>
+        <div class="oe_span6">
+            <div class="oe_demo oe_picture oe_screenshot">
+                <img src="screenshot-2.png"/>
+            </div>
+        </div>
+    </div>
+</section>
+
+<section class="oe_container oe_dark">
+    <div class="oe_row oe_spaced">
+        <div class="oe_span6">
+            <p class="oe_mt32">
+                If it does then a cashier get a popup with selection users.
```

Comment:
If it does and cashier doesn't have right to approve it, then a popup will be shown.